modules/pam_faillock/pam_faillock.8.xml

```diff
@@ -327,6 +327,12 @@ session  required       pam_selinux.so open
         <term><filename>/var/run/faillock/*</filename></term>
         <listitem>
           <para>the files logging the authentication failures for users</para>
+          <para>
+            Note: These files will disappear after reboot on system’s configured with
```

Comment:
Likewise, it's `on systems`, not `on system’s`.

Reply:
Hi Dmitry,

Thank you for your feedback.

I have made the necessary changes.

Can you kindly review again.

Thank you!